go.mod

```diff
@@ -39,10 +39,10 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.3 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/emicklei/go-restful/v3 v3.11.0 // indirect
-	github.com/evanphx/json-patch v5.6.0+incompatible // indirect
+	github.com/evanphx/json-patch v4.12.0+incompatible // indirect
```

Comment:
this downgrade was interesting, but it matches upstream k8s  https://github.com/kubernetes/kubernetes/blob/release-1.29/go.mod#L34